app/lib/constants/site_config.rb

```diff
@@ -143,6 +143,14 @@ module SiteConfig
         description: "Special cute mascot image used in the footer.",
         placeholder: "https://image.url"
       },
+      mascot_footer_image_width: {
+        description: "The footer image will be resized to this value, defaults to 52",
```

Comment:
Shouldn't that say something like:

> The footer **mascot width** will be resized to this value, defaults to 52

?

Same for the height couple lines below

Reply:
Yes, it could be either, since this is the "mascot image." I think your suggestion is more coherent, so I'll make that change!